File: comparison/results/haiku/baseline/humaneval/HumanEval_36/java/solution.java
public static long fizzBuzz(long n) {
    long count = 0;
    for (long i = 1; i < n; i++) {
        if (i % 11 == 0 || i % 13 == 0) {
            long temp = i;
            while (temp > 0) {
                if (temp % 10 == 7) {
                    count++;
                }
                temp /= 10;
            }
        }
    }
    return count;
}

File: comparison/results/haiku/baseline/humaneval/HumanEval_36/javascript/solution.js
function fizz_buzz(n){
  let count = 0;
  for (let i = 1; i < n; i++) {
    if (i % 11 === 0 || i % 13 === 0) {
      count += String(i).split('').filter(digit => digit === '7').length;
    }
  }
  return count;
}
